lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -1202,9 +1243,9 @@ def seek(self, offset: int, _whence: int = io.SEEK_SET) -> int:
                 return offset
 
         # Invoke the CUT
-        stream = Inventory(MockBufferedReader(), None)
+        stream = Inventory(MockBufferedReader())
 
-        assert stream.subproc is None
+        assert stream.lock is None
```

Comment:
Shouldn't both `lock` and `subproc` be `None`?  (It seems reasonable to check them both.)

Ditto at line 1455.